ruma-events/tests/enums.rs

```diff
@@ -243,3 +244,24 @@ fn alias_event_deserialization() {
         if aliases == vec![ RoomAliasId::try_from("#somewhere:localhost").unwrap() ]
     );
 }
+
+#[test]
+fn alias_event_field_access() {
+    let json_data = aliases_event();
+
+    assert_matches!(
+        from_json_value::<AnyEvent>(json_data.clone()),
+        Ok(AnyEvent::State(state_event))
+        if state_event.state_key() == ""
+            && state_event.room_id() == &RoomId::try_from("!room:room.com").unwrap()
+            && state_event.event_id() == &EventId::try_from("$152037280074GZeOm:localhost").unwrap()
```

Comment:
Not sure if a sanity check test for the access methods is worth it?

Reply:
Sure, better than no test :)